src/renderer/components/InputCurrency.js

```diff
@@ -116,8 +118,9 @@ class InputCurrency extends PureComponent<Props, State> {
     }
   }
 
-  handleChange = (v: string) => {
-    const { onChange, unit, value, locale } = this.props;
+  handleChange = (val: string) => {
+    const { onChange, unit, value, locale, decimals } = this.props;
+    const v = decimals === 0 ? val.replace(/[.,]/g, "") : val;
```

Comment:
mmh this is covered by the sanitizeValueString for a unit that is decimal=0. can you explain the usecase here?